src/Microsoft.VisualStudio.Editors/DesignerFramework/AccessModifierComboBox.vb

```diff
@@ -354,48 +354,30 @@ Namespace Microsoft.VisualStudio.Editors.DesignerFramework
         Protected Function GetMenuCommandsToRegister(commandIdCombobox As CommandID, commandIdGetDropdownValues As CommandID) As ICollection
             ' For a dynamic combobox, we need to add two commands, one to handle the combobox, and one to fill
             ' it with items...
-            Dim MenuCommands As New List(Of MenuCommand)
-            _designerCommandBarComboBoxCommand = New DesignerCommandBarComboBox(_rootDesigner, commandIdCombobox, AddressOf GetCurrentValue, AddressOf SetCurrentValue, AddressOf EnabledHandler)
+            Dim menuCommands As New List(Of MenuCommand)
+            _designerCommandBarComboBoxCommand = New DesignerCommandBarComboBox(_rootDesigner, commandIdCombobox, AddressOf GetCurrentAccessibilityValue, AddressOf SetCurrentAccessibilityValue, AddressOf EnabledHandler)
             _commandIdCombobox = commandIdCombobox
-            MenuCommands.Add(_designerCommandBarComboBoxCommand)
-            MenuCommands.Add(New DesignerCommandBarComboBoxFiller(_rootDesigner, commandIdGetDropdownValues, AddressOf GetDropdownValues))
+            menuCommands.Add(_designerCommandBarComboBoxCommand)
+            menuCommands.Add(New DesignerCommandBarComboBoxFiller(_rootDesigner, commandIdGetDropdownValues, AddressOf GetDropdownValues))
 
             RegisterMenuCommandForwarder()
 
-            Return MenuCommands
-        End Function
-
-        ''' <summary>
-        ''' Tries to retrieve the value of the "Custom Tool" property.  If there is no such
-        '''   property in this project, returns False.
-        ''' </summary>
-        ''' <param name="value"></param>
-        Private Function TryGetCustomToolPropertyValue(ByRef value As String) As Boolean
-            value = Nothing
-
-            Dim ToolProperty As EnvDTE.Property = DTEUtils.GetProjectItemProperty(_projectItem, DTEUtils.PROJECTPROPERTY_CUSTOMTOOL)
-            If ToolProperty IsNot Nothing Then
-                Dim CurrentCustomToolValue As String = TryCast(ToolProperty.Value, String)
-                value = CurrentCustomToolValue
-                Return True
-            End If
-
-            Return False
+            Return menuCommands
         End Function
 
         ''' <summary>
         ''' Return the current accessibility value
         ''' </summary>
-        Private Function GetCurrentValue() As String
+        Private Function GetCurrentAccessibilityValue() As String
             Dim currentValue As String
-            Dim matchingEntry As CodeGenerator = GetCurrentMatchingGenerator()
-            If matchingEntry IsNot Nothing Then
-                currentValue = matchingEntry.DisplayName
+            Dim matchingCodeGenerator As CodeGenerator = GetCurrentMatchingGenerator()
```

Comment:
To all the rest of the use of `codeGenerator` vs `entry`: I don't really understand what an 'entry' is, but why else would you be adding/iterating a list of 'entries'? If they aren't an 'entry', maybe the `_codeGeneratorEntries` variable should be changed too.

Reply:
Maybe a _codeGeneratorList.

My understanding is that entry might be an input from the user.

I will leave this name for now and see if later it makes sense 'Entry'.